search/src/Components/dataset/dataset_summary.js

```diff
@@ -303,6 +323,8 @@ export const DatasetSummary = (props) => {
     const datasetUrl = "https://weed-ai.sydney.edu.au/datasets/" + upload_id;
     displayMeta["identifier"] = [...displayMeta.identifier || [], datasetUrl];
     const authorList = listToWords(displayMeta.creator.map((creator) => creator.name));
+    const head = parseInt(head_version);
+    const old_versions = isNaN(head) ? [] : [...Array(head - 1).keys()].map((i) => i + 1).reverse();
```

Comment:
I thought this was retrieving available previous versions. So this is assuming that we will have all the versions available till the head version.